tests/test_main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import pytest
+from app.main import is_isogram
+
+
+@pytest.mark.parametrize(
+    "word, boolean_value",
+    [
+        ('pythonist', False),
+        ('must', True),
+        ('be', True),
+        ('lazy', True),
+        ('alwAys', False),
+        ('anD', True),
+        ('EvERywhEre', False),
+    ],
+)
```

Comment:
Add `pytest.param( [], [], id="Some description" )` to mark what case you are testing.